xds/internal/server/rds_handler.go

```diff
@@ -21,6 +21,7 @@ package server
 import (
 	"sync"
 
+	internalgrpclog "google.golang.org/grpc/internal/grpclog"
```

Comment:
Optional: internalgrpclog is long. I feel like usually I see this as igrpclog elsewhere in codebase.

Reply:
Done.